source/api_documentation/network_integration/call_recording_condition/index.rst

```diff
@@ -0,0 +1,84 @@
+Call Recording Condition
+========================
+
+Manage call recording conditions for campaigns
+""""""""""""""""""""""""""""""""""""""""""""
+
+A call recording condition can be set for a campaign to determine whether or not the call should be recorded. The call will be recorded if the full condition is met.
+
+Set Call Recording Condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To conditionally record calls, set ``"record_calls"`` to ``true`` and set ``"call_recording_condition"`` to the expression you want to evaluate.
+
+POST
```

Comment:
Should PUT examples be included as well? Or are would that be overkill considering the advertiser campaigns section of the NIAPI docs list examples for all of the relevant HTTP verbs?

Reply:
I'd say it's not worth it